packages/emails/email-manager.ts

```diff
@@ -285,3 +286,25 @@ export const sendDisabledAppEmail = async ({
 }) => {
   await sendEmail(() => new DisabledAppEmail(email, appName, appType, t, title, eventTypeId));
 };
+
+export const sendDailyVideoRecordingEmails = async (calEvent: CalendarEvent, downloadLink: string) => {
+  const emailsToSend: Promise<unknown>[] = [];
+
+  emailsToSend.push(
+    ...calEvent.attendees.map((attendee) => {
+      return new Promise((resolve, reject) => {
```

Comment:
Let's avoid this repetitive Promise creation and try catch by using `sendEmail` method e.g. see https://github.com/calcom/cal.com/blob/d6e3f2fe91668b52e2b2b86888f2963c5e253b61/packages/emails/email-manager.ts#L55